### DISCUSSION TITLE
Does the Pico Scoppy oscilloscope support single shot trigger?

### DISCUSSION THREAD

**Discussion by orientzane** [fhdm-dev/scoppy#126] [Q&A]
Hello:
Does the Pico Scoppy oscilloscope support single shot trigger?
Thanks
![a66e7261dd186a75532134ffefc8fbb2,w_800](https://github.com/fhdm-dev/scoppy/assets/38416895/0652fe2d-041d-414e-ba1f-c2ecd29ac94f)

**Comment by fhdm-dev** [comment]
Yes.

Please note that single capture mode uses the same trigger settings as run mode so you need to set the trigger to NORM to prevent the single capture from starting before a trigger event actually occurs. I might change this behavior in a future version of the app so that it functions more like a real oscilloscope.
